config/locales/en.yml

```diff
@@ -34,7 +34,6 @@ en:
             invalid_requirements_types:
               one: 'requirements.json contains an invalid type: %{invalid_types}'
               other: 'requirements.json contains invalid types: %{invalid_types}'
-            multiple_channel_integrations: 'Specifying multiple channel integrations in requirements.json is not supported.'
```

Comment:
Looks like @timbertson [added this string](https://github.com/zendesk/zendesk_apps_support/commit/fc5493c5beaf54e457f7d5643e1b81abd9db0f51) directly here, we should copy it to zendesk_apps_support.yml instead of delete it.